src/StripeMethod.ts

```diff
@@ -1,7 +1,5 @@
-'use strict';
-
-const utils = require('./utils');
-const makeRequest = require('./makeRequest');
+import * as utils from './utils';
```

Comment:
How to we decide which types to rename to `.ts` and which to keep as `.js`?

Reply:
The goal is to eventually change everything to `.ts` - this PR starts by migrating the most important Stripe infrastructure files. The remaining infrastructure will be migrated next, followed by generated files and tests.